docs/src/openapi-v1.yaml

```diff
@@ -962,6 +962,114 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /pallets/{palletId}/errors:
+    get:
+      tags:
+      - pallets
+      summary: Get a list of errors for a pallet.
+      description: Returns a list of error item metadata for error items of the
+        specified palletId.
+      parameters:
+      - name: palletId
+        in: path
+        description: 'Name or index of the pallet to read error metadata for. Note: the pallet name must match what is
```

Comment:
Why is this wrapped in quotes?

Reply:
I think this is wrapped in quotes because of the colon after `Note`. I saw this is how its also formatted for other sections in the yaml that have a colon in the text.

Reply:
Ahh, cool I didnt notice that. The Quotes can stay!